llvm/test/SYCLLowerIR/printf_addrspace/generic_as_no_opt.ll

```diff
@@ -6,8 +6,14 @@
 ;; SYCLMutatePrintfAddrspacePass turned off):
 ;; clang++ -fsycl -fsycl-device-only Inputs/experimental-printf.cpp -S -O0 -D__SYCL_USE_NON_VARIADIC_SPIRV_OCL_PRINTF__
 
-; RUN: opt < %s --SYCLMutatePrintfAddrspace -S | FileCheck %s --implicit-check-not=obsolete
-; RUN: opt < %s --SYCLMutatePrintfAddrspace -S --enable-new-pm=1 | FileCheck %s --implicit-check-not=obsolete
+; RUN: opt < %s --SYCLMutatePrintfAddrspace -S | FileCheck %s
+; RUN: opt < %s --SYCLMutatePrintfAddrspace -S --enable-new-pm=1 | FileCheck %s
+
+; Check that the wrapper bodies have been deleted after call replacement
+; CHECK-NOT: spir_func i32 @{{.*}}sycl{{.*}}printf
+; Make sure the non-variadic declarations have been wiped out
+; in favor of the single variadic one:
+; CHECK-NOT: declare dso_local spir_func i32 @_Z18__spirv_ocl_printf{{.*}}(i8 addrspace(4)*, {{[:alnum:]+}})
```

Comment:
Am I right that those can be removed in favor of `generic_as_negative_checks.ll` test?

Reply:
Yep, I've left these over by accident and subsequently removed them via https://github.com/intel/llvm/pull/5069/commits/1e3493e310aad78eb28568321ffc6d723b5ea553.